formats/statistics_announcement/publisher/details.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "release_date",
+    "state"
+  ],
+  "properties": {
+    "cancellation_reason": {
+      "type": "string"
+    },
+    "cancelled_at": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "latest_change_note": {
+      "type": "string"
+    },
+    "logo": {
+      "$ref": "#/definitions/image"
+    },
+    "previous_display_date": {
+      "type": "string"
+    },
+    "release_date": {
```

Comment:
Whats the reason for this (and `previous_display_date`) being a string, rather than a `datetime` type?